ssl/t1_lib.c

```diff
@@ -3218,9 +3218,8 @@ int tls_choose_sigalg(SSL *s, int fatalerrs)
 
                 /* For Suite B need to match signature algorithm to curve */
                 if (tls1_suiteb(s))
-                    curve =
-                        evp_pkey_get_EC_KEY_curve_nid(s->cert->pkeys[SSL_PKEY_ECC]
-                                                      .privatekey);
+                    curve = ssl_get_EC_curve_nid(s->cert->pkeys[SSL_PKEY_ECC]
```

Comment:
Here it is in OPENSSL_NO_EC as well but probably does not need to anymore.

Reply:
Done, although I'm a little unsure about the consequences...